ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTracker.java

```diff
@@ -66,7 +66,7 @@ public static void trackChainHeadForPeers(
 
   @Override
   public void onPeerConnected(final EthPeer peer) {
-    LOG.debug("Requesting chain head info for {}", peer);
+    LOG.debug("Requesting chain head info from {}", peer);
```

Comment:
here we use "info" but below we use "information" - might be good to make consistent

Reply:
uniformed to use shorter form "info"